client/package.json

```diff
@@ -49,9 +49,11 @@
     "karma-coverage-istanbul-reporter": "^2.0.4",
     "karma-jasmine": "~1.1.1",
     "karma-jasmine-html-reporter": "^1.3.1",
+    "mini-css-extract-plugin": "^0.4.4",
     "protractor": "^5.4.1",
     "ts-node": "^7.0.1",
     "tslint": "^5.11.0",
-    "typescript": "^2.9.2"
+    "typescript": "^2.9.2",
+    "webpack": "^4.21.0"
```

Comment:
why did you add the webpack and mini css extract plugin dependency? currently we relay on the angular cli without an extra webpack config for building I believe?

Reply:
Yes you are right, but this was required when I used to develop on windows. But now in the latest commit I have removed it.